apps/demo-app/src/app/examples/search-filter/search-filter/search-filter-example.component.ts

```diff
@@ -36,7 +36,7 @@ export class SearchFilterExampleComponent {
     this.users = this._usersService.getUsers(searchParams);
   }
 
-  private createSearchFilters(): SearchFilterBase<any>[] {
+  private createSearchFilters(): SearchFilterBase<unknown>[] {
```

Comment:
yet another type in typescript, bah

Reply:
unknown is consider a "type-safe" alternative to any

https://www.totaltypescript.com/concepts/object-is-of-type-unknown

With any, you can access any property on it without narrowing it.
This is not possible with unknown

Reply:
We should forbid usage of any. No point of using TypeScript (typed JS) if you prefer using any. You can sort out 99.99% of cases with unknowns. It's been there for years. You can still use nice features of JS such as accessing properties in objects in a dictionary way with use of Record<> and other rich type features added in versions 4.x